aws/aws.go

```diff
@@ -39,6 +62,46 @@ func GetAllResources() (*AwsAccountResources, error) {
 
 		resourcesInRegion := AwsRegionResource{}
 
+		// ASG Names
+		groupNames, err := getAllAutoScalingGroups(session, region)
```

Comment:
You might want to add a comment near the top of this method that the order of these resources matters due to dependencies between them: e.g., ASGs first, then EC2 instances, then EBS volumes.